### PR TITLE
Fixed the representation of conversion equations of true anomaly

### LINKED ISSUE #1093: Wrong Equations in API Documentation
Hello, 
While I was traversing the documentation, I found that equations stated here [1](https://docs.poliastro.space/en/latest/autoapi/poliastro/core/angles/index.html#poliastro.core.angles.E_to_nu) & [2](https://docs.poliastro.space/en/latest/autoapi/poliastro/core/angles/index.html#poliastro.core.angles.F_to_nu) are wrong (discussed with @astrojuanlu)

🐞 **Problem**

The equation here is not correct & hence can be fixed with the correct representation {Latex}. 

🖥 **Please paste the output of following commands**

* `conda info -a` (only if you have conda)
* `conda list` (only if you have conda)
* `pip freeze`

<details>
  
```bash
# Paste your output here:
Not Required 🤔 

```
</details>

<details>

```bash
# Paste your output here:
Not Required 🤔 
```
</details>

💡 **Possible solutions**
 

Representation of the equation can be altered as to demonstrate the correct equation 😄 
📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by DhruvSondhi**
This PR aims to fix #1093 
Changes to all major representation of anomaly conversion 😄

### CODE REVIEW

File: src/poliastro/core/angles.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def nu_to_E(nu, ecc):
     The implementation uses the half-angle formula from [3]_:
 
     .. math::
-        E = 2 \atan \left( \sqrt{\frac{1 - e}{1 + e}} \tan{\frac{\nu}{2}}
+        E = 2 \operatorname{arctan} \left ( \sqrt{\frac{1 - e}{1 + e}} \tan{\frac{\nu}{2}} \right)
```
Comment (astrojuanlu) [inline src/poliastro/core/angles.py:152]: Don't we have `\arctan` in LaTeX? Can you give it a try locally?

**Review by astrojuanlu — COMMENTED** [review]
Thanks @DhruvSondhi ! Left a very small comment, please check before proceeding.

File: src/poliastro/core/angles.py (hunk 1 above)
Comment (DhruvSondhi) [reply src/poliastro/core/angles.py:152]: Sure ... checking & confirming the same

Comment (DhruvSondhi) [reply src/poliastro/core/angles.py]: Yes does work ... changed `\operatorname{arctan}` to `\arctan`

Comment (DhruvSondhi) [reply src/poliastro/core/angles.py]: For reference ... Inverse Hyperbolic Tangent Function does need `\operatorname{arctanh}` for parsing into Latex :)

**Comment by astrojuanlu** [comment]
Merged, thanks!

**Pull request merged**